sdk/doc/generate.go

```diff
@@ -99,3 +108,222 @@ func genMarkdownTreeCustom(cmd *cobra.Command, rootdir string, filePrepender, li
 	cmd.DisableAutoGenTag = true
 	return doc.GenMarkdownCustom(cmd, f, linkHandler)
 }
+
```

Comment:
could you add an example of how we have to write doc now ?

Reply:
done, cf. https://github.com/ovh/cds/pull/2012/commits/778af69839075f610e779b8f4595ac5ed964ea20